### ISSUE TITLE
Is this task designed for on-premise too?

### ISSUE THREAD

**Issue by Tomers12345** [redhat-developer/openshift-vsts#150]
I downloaded OpenShift extension v1.5.0 from  Azure DevOps Marketplace ,
and installed this task on premise Azure Devops 2019 .
but i dont see OpenShift service connection under "New service connection"

**Comment by lstocchi** [comment]
Hi @Tomers12345 , i never tried on a on-premise Azure Devops 2019 version but it should work the same. Can you please confirm that the extension is installed correctly (under manage extensions)? Is it just the OpenShift service connection you don't see? Are you able to see the different tasks when building a pipeline (i.e Execute oc command)? 
Thank you

**Comment by Tomers12345** [comment]
Hi ,
The extension is installed correctly(called OpenShift Extension)
I dont see any oc task either

**Comment by lstocchi** [comment]
@Tomers12345 I tried installing Azure Devops 2019 server on my machine and installing OpenShift extension but I didn't have any problem. I can correctly find it under service connections and tasks. Any other info you could give me to replicate it? Atm i don't have any clue about why this is happening to you.

**Comment by lstocchi** [comment]
@Tomers12345 have you tried by using different browsers? by reading around I found that there were other users with the same issue. 

- https://developercommunity.visualstudio.com/content/problem/398389/installed-pipeline-task-cant-be-added.html 
- https://developercommunity.visualstudio.com/content/problem/394821/i-cannot-add-a-task-that-i-downloaded-from-the-azu.html
- https://developercommunity.visualstudio.com/content/problem/367509/all-the-build-pipeline-extensions-installed-throug.html

Developers from Microsoft says this was solved but some user is still facing it. Hopefully this will be solved by waiting. Not a good solution but I don't know what else to suggest you right now. Let me know if the problem fixes itself overnight
